monai/fl/client/monai_client.py

```diff
@@ -88,8 +91,213 @@ def disable_ckpt_loaders(parser):
                     h["_disabled_"] = True
 
 
+def compute_total_stats(case_stats_lists, hist_bins, hist_range):
+    # Compute total summary
+    total_case_stats = []
+    for case_stats_list in case_stats_lists:
+        total_case_stats += case_stats_list
+
+    for feature_name in hist_bins:
+        hist_bins = hist_bins[feature_name]
+        hist_range = hist_range[feature_name]
+
+    summarizer = SegSummarizer("image", "label", average=True, do_ccp=True,
+                               hist_bins=hist_bins, hist_range=hist_range)
+    total_summary_stats = summarizer.summarize(total_case_stats)
+
+    summary_stats = {
+        FlStatistics.DATA_STATS: total_summary_stats,
+        FlStatistics.DATA_COUNT: len(total_case_stats),
+        FlStatistics.FAIL_COUNT: 0,
+        FlStatistics.FEATURE_NAME: "Intensity"
+    }
+
+    return summary_stats
+
+
+class MonaiStats(ClientStats):
```

Comment:
I think `MonaiStats` sounds like too general a class name, and within the MONAI codebase, why we call it "MONAI XXX"?

Thanks.

Reply:
Okay, I went with `ClientAlgoStats` and `MonaiAlgoStats`. Better?